# Dia 2: JavaScript ES6 - Fluxo de exceção e Objetos

Hoje nós aprendemos sobre fluxo de exceção, como lançar erros com o operador `throw`, como tratar de erros sem parar a aplicação através de blocos `try/catch` e novos métodos para acessar objetos como o `Object.keys()`, `Object.values`, `Object.entries` e `Object.assign`

## Exercícios desenvolvidos

### Parte I
